examples/evm_transaction_ecdsa.js

```diff
@@ -0,0 +1,241 @@
+
+    const init = [
+      // this parameter is mandatory for all init arrays. Note we use the ETH address here(!)
+      {
+        vname: '_evm_version',
+        type: 'Uint32',
+        value: '0',
+      },
+      {
+        vname: 'owner',
+        type: 'ByStr20',
+        value: `${zilliqa.wallet.defaultAccount.address}`
+      },
+    ];
+
+    // Instance of class Contract
+    const contract = zilliqa.contracts.new(code, init);
+
+    console.log("Deploying contract proper.");
+
+    // Deploy the contract.
+    // Also notice here we have a default function parameter named toDs as mentioned above.
+    // A contract can be deployed at either the shard or at the DS. Always set this value to false.
+    const [deployTx, hello] = await contract.deploy(
+      {
+        version: VERSION_UPGRADE,
+        gasPrice: myGasPrice,
+        gasLimit: Long.fromNumber(6000000),
+      },
+      33,
+      1000,
+      false,
+    );
+
+    // Introspect the state of the underlying transaction
+    console.log(`Deployment Transaction ID: ${deployTx.id}`);
+    console.log(`Deployment Transaction Receipt:`);
+    console.log(deployTx.txParams.receipt);
+
+    // Get the deployed contract address
+    console.log('The contract address is:');
+    console.log(hello.address);
+    //Following line added to fix issue https://github.com/Zilliqa/Zilliqa-JavaScript-Library/issues/168
+    const deployedContract = zilliqa.contracts.at(hello.address);
+
+    // Create a new timebased message and call setHello
+    // Also notice here we have a default function parameter named toDs as mentioned above.
+    // For calling a smart contract, any transaction can be processed in the DS but not every transaction can be processed in the shards.
+    // For those transactions are involved in chain call, the value of toDs should always be true.
+    // If a transaction of contract invocation is sent to a shard and if the shard is not allowed to process it, then the transaction will be dropped.
+    const newMsg = 'Hello, the time is ' + Date.now();
+    console.log('Calling setHello transition with msg: ' + newMsg);
+    const callTx = await hello.call(
```

Comment:
Does this work with an EVM contract?   The meesage above says it is calling a transition, assuming Scilla?